sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -357,6 +402,9 @@ export const enum KnownInnerErrorCodeValue {
     UnsupportedLanguageCode = "UnsupportedLanguageCode"
 }
 
+// @public
+export type KnownSentencesSortBy = "Offset" | "Rank";
```

Comment:
Rank -> Importance

Reply:
I worry people won't connect rankScore with Importance...

Reply:
I renamed rankScore to importanceScore as well

Reply:
reverted back until the service changes it on their side.